modules/outconBidAdapter.js

```diff
@@ -0,0 +1,41 @@
+import {registerBidder} from '../src/adapters/bidderFactory';
+import {config} from '../src/config';
+const BIDDER_CODE = 'outcom';
+export const spec = {
+  code: BIDDER_CODE,
+  aliases: ['outcom'],
```

Comment:
If you have the same name as the value for your BIDDER_CODE, this `alias` property doesn't need to be defined.  This should only be used when you have a different name.